File: cpp-test-files/46.cpp
//switch
#include <iostream>

int main() {
    int number = 2;

    switch (number) {
        case 1:
            std::cout << "You selected one." << std::endl;
            break;
        case 2:
            std::cout << "You selected two." << std::endl;
            break;
        case 3:
            std::cout << "You selected three." << std::endl;
            break;
        default:
            std::cout << "Invalid selection!" << std::endl;
    }

    return 0;
}

File: java-files/46.java
public class SwitchExample {
    public static void main(String[] args) {
        int number = 2;
        switch (number) {
            case 1:
                System.out.println("You selected one.");
                break;
            case 2:
                System.out.println("You selected two.");
                break;
            case 3:
                System.out.println("You selected three.");
                break;
            default:
                System.out.println("Invalid selection!");
        }
    }
}